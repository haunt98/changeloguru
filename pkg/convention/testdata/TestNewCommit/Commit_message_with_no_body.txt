docs: correct spelling of CHANGELOG
