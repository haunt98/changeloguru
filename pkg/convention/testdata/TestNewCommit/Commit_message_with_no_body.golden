{
  "RawHeader": "docs: correct spelling of CHANGELOG",
  "Type": "docs"
}